tox.ini

```diff
@@ -1,43 +1,20 @@
 [tox]
 envlist =
-    py{36,37,38}-test
+    test
     build_docs
-    codestyle
 isolated_build = true
-# This is included for testing of the template. You can remove it safely.
-skip_missing_interpreters = True
 
 [testenv]
-# Pass through the following environment variables which may be needed for the CI
-passenv = HOME, WINDIR, LC_ALL, LC_CTYPE, CC, CI, TRAVIS
-
-# Suppress display of matplotlib plots generated during docs build
-setenv = MPLBACKEND=agg
-
-# Run the tests in a temporary directory to make sure that we don't import
-# the package from the source tree
 changedir = .tmp/{envname}
-
-# tox environments are constructed with so-called 'factors' (or terms)
-# separated by hyphens, e.g. test-devdeps-cov. Lines below starting with factor:
-# will only take effect if that factor is included in the environment name. To
-# see a list of example environments that can be run, along with a description,
-# run:
-#
-#     tox -l -v
-#
-description =
-    run tests
-
 deps =
-# The following indicates which extras_require from setup.cfg will be installed
+    # TMP: should be changed once changes are in astropy
+    git+https://github.com/astrofrog/astropy@iers-package
 extras =
     test
-    alldeps: all
 
 commands =
     pip freeze
-    pytest --pyargs astropy_iers_data {toxinidir}/docs --cov astropy_iers_data --cov-config={toxinidir}/setup.cfg {posargs}
+    pytest --pyargs astropy {posargs}
```

Comment:
Can we run tests only for some subpackages ( I guess `utils.iers` and `coordinates`) ?

Reply:
I thought of this but we also have to include astropy.time and then I was thinking well why not just run it all as in case by chance we ever add usage of IERS elsewhere in the core package? It will only be running weekly so maybe not a major CI usage?